pages/_app.js

```diff
@@ -1,10 +1,39 @@
 import GlobalStyle from "../styles";
+import useLocalStorageState from "use-local-storage-state";
+import { plants } from "@/assets/plants";
+import Layout from "@/components/Layout";
+import { uid } from "uid";
+const initialPlants = plants;
 
 export default function App({ Component, pageProps }) {
+  const [plants, setPlants] = useLocalStorageState("favourites", {
```

Comment:
Why is the local storage called 'favourites'??? Since it is the plants you should name it like this.